doc/classes/Line2D.xml

```diff
@@ -210,44 +213,63 @@
 	</methods>
 	<members>
 		<member name="begin_cap_mode" type="int" setter="set_begin_cap_mode" getter="get_begin_cap_mode" enum="Line2D.LineCapMode">
+			The style used for the shape in the first point of the line.
 		</member>
 		<member name="default_color" type="Color" setter="set_default_color" getter="get_default_color">
+      The color of the Line2D. Will not be used if a gradient is set.
 		</member>
 		<member name="end_cap_mode" type="int" setter="set_end_cap_mode" getter="get_end_cap_mode" enum="Line2D.LineCapMode">
+			The style used for the shape in the last point of the line.
```

Comment:
Too many prepositions. Instead of "foo of the bar" just use "bar's foo".
Typically, it's helpful to note the default value. 
Try something like:

    The final point's style. Use [code]LINE_CAP_*[/code] constants. Default value: [code]LINE_CAP_NONE[/code].